content/posts/fpcast.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Pyodide Function Pointer Cast Removal"
+date: 2021-12-06T20:27:27-08:00
+draft: true
+tags: ["internals"]
+author: "Hood Chatham"
+# author: ["Me", "You"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Why old versions of Pyodide had a low recursion limit and how version 0.19.0 supports a much higher one."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    image: "<image path/url>" # image path/url
+    alt: "<alt text>" # alt text
+    caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+In Pyodide v0.19, we can finally support the Python default recursion limit of
+1000. We also attain a speed up between 8% to 20% for loading Pyodide and for
+individual benchmarks. The entire test suite runs faster by a comparable amount.
+The code size was reduced by 25% from 12 megabytes to 9.1 megabytes. These gains
```

Comment:
Maybe,
```suggestion
The code size was reduced by 25% from 12 MB to 9.1 MB. These gains
```
unless  you think the abbreviation is unclear to the target audience.

Reply:
I think for the third sentence I like it a tiny bit better spelled out. I searched around and found the following:

> most style guides recommend writing out units of measure (e.g., the word feet instead of the abbreviation ft.) unless you are writing a technical or scientific document, but abbreviations are always acceptable in figures and tables.

Which of course doesn't mean anything. But it justifies the preference I already had. =)